ui/packages/codice-icons/codice.font.css

```diff
@@ -0,0 +1,3 @@
+.codice:before {
+  vertical-align: middle !important;
```

Comment:
❓ What is this used for?

Reply:
The webfonts-loader had the `vertical-align` set to `top` which made the custom icons unaligned with the rest of the icons, so this just overrides that so that it matches the icons we are already using.